.claude/commands/ship.md

```
@@ -0,0 +1,95 @@
+
+## 3단계: 커밋
+
+관련 파일만 명시적으로 스테이징한다 (.env, secrets 절대 포함 금지):
+
+```bash
+git add <관련 파일들>
+```
+
+커밋 메시지는 Conventional Commits + 이슈 번호 형식을 반드시 따른다:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)